docker/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.10
+
+ARG VERSION=0.1.1
+
+LABEL name="Boundary" \
+      maintainer="HashiCorp Boundary Team <boundary@hashicorp.com>" \
+      vendor="HashiCorp" \
+      version=$VERSION \
+      release=$VERSION \
+      summary="Boundary provides simple and secure access to hosts and services" \
+      description="The Boundary Docker image is designed to enable practitioners to run Boundary in server mode on a container scheduler"
+
+RUN addgroup boundary && \
+    adduser -s /bin/sh -S -G boundary boundary
+
+ADD https://releases.hashicorp.com/boundary/${VERSION}/boundary_${VERSION}_linux_amd64.zip /tmp/
+ADD https://releases.hashicorp.com/boundary/${VERSION}/boundary_${VERSION}_SHA256SUMS /tmp/
+ADD https://releases.hashicorp.com/boundary/${VERSION}/boundary_${VERSION}_SHA256SUMS.sig /tmp/ 
+
+RUN apk add --no-cache ca-certificates gnupg openssl libcap su-exec dumb-init tzdata 
+RUN cd /tmp/ && \
+    BUILD_GPGKEY=91A6E7F85D05C65630BEF18951852D87348FFC4C; \
+    found=''; \
+    for server in \
+        hkp://p80.pool.sks-keyservers.net:80 \
+        hkp://keyserver.ubuntu.com:80 \
+        hkp://pgp.mit.edu:80 \
+    ; do \
```

Comment:
I would probably just hardcode the key in this repo somewhere and import it, rather than be exposed to the internet to download it, since its fingerprint is already hardcoded. Not blocking though.

Reply:
Totally hear that, I was torn between having consistency between our other Dockerfiles though.